test/server/cards/03_TWI/leaders/GeneralGrievousGeneralOfTheDroidArmies.spec.ts

```diff
@@ -0,0 +1,83 @@
+describe('General Grievous, General Of The Droid Armies', function () {
+    integration(function (contextRef) {
+        describe('General Grievous\'s leader undeployed ability', function () {
+            it('should Sentinel to a Droid unit for this phase', function () {
```

Comment:
nit: since there's only `it()` in this `describe()` we can just remove the `describe` completely and put the whole name inside of the `it`, same for the other test